src/PKSim.Infrastructure/Serialization/ORM/Mappers/ProjectToProjectMetaDataMapper.cs

```diff
@@ -164,11 +165,39 @@ public void Visit(Compound compound)
 
       public void Visit(Individual individual)
       {
-         _metaData = new IndividualMetaData();
+         _metaData = new IndividualMetaData
+         {
+            ExpressionProfileIds = expressionProfileIdsFor(individual)
+         };
          serializeContentFor(individual);
          _metaData.Properties.Data = _serializationManager.Serialize(individual.OriginData);
       }
 
+      public void Visit(RandomPopulation randomPopulation)
+      {
+         _metaData = new RandomPopulationMetaData
+         {
+            ExpressionProfileIds = expressionProfileIdsFor(randomPopulation)
+         };
+         //no need to compress population data. This is mostly only double arrays
+         serializeContentFor(randomPopulation, compress: false);
+         _metaData.Properties.Data = _serializationManager.Serialize(randomPopulation.Settings);
+      }
+
+      public void Visit(ImportPopulation importPopulation)
+      {
+         _metaData = new ImportPopulationMetaData
+         {
+            ExpressionProfileIds = expressionProfileIdsFor(importPopulation)
```

Comment:
@Yuri05 I am saving the ids used in the DB as discussed. That way, I can load the bb on demand when required